.github/workflows/maven-ubuntu-linux.yml

```diff
@@ -30,10 +30,15 @@ jobs:
       run: mvn -B package
     - name: Copy JAR to staging
       run: cp target/*.jar staging
-    - name: Build and package as Linux application with Gluon
-      run: mvn client:build client:package
-      env:
-        GRAALVM_HOME: /opt/hostedtoolcache/GraalVM/20.2.0-java11/x64
+    - name: Gluon Build
+      uses: gluonhq/gluon-build@v1
+      with:
+        gluon-license: ${{ secrets.GLUON_LICENSE }}
+        target: 'linux'
```

Comment:
use 'desktop' here. that will build for the 'host' client target, which is linux